index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio | Home Page</title>
+    <link rel="stylesheet" href="./styles.css">
+    <script src="https://kit.fontawesome.com/59ba0ac1be.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <h3 class="logo-text">My Logo</h3>
+    <nav>
+      <ul>
+        <li><a href="./portfolio.html">Portfolio</a></li>
+        <li><a href="./about.html">About</a></li>
+        <li><a href="./contact.html">Contact</a></li>
+      </ul>
+    <i class="fa-sharp fa-solid fa-bars"></i>
+    </nav>
+  </header>
+  <section id="introduction">
+    <h1 class="bigtext1">I'm Alexandra <br> Glad to see you!</h1>
+    <h1 class="bigtext2">I'm Mariane <br> Glad to see <br>you!</h1>
```

Comment:
- [ ] [OPTIONAL] There should be only one `h1` in an HTML file because of semantics and it helps search engines and web crawlers easily understand the main subject of your page. I kindly suggest that you make use of one h1 per html page.

Reply:
@topeogunleye 
Thank you so much. This too has been fixed.